src/compiler/Stage/Parse/Parser.elm

```diff
@@ -410,23 +410,74 @@ literalInt =
         |> P.inContext InLiteralInt
 
 
-{-| TODO escapes
-TODO Unicode escapes
--}
+-- for literalChar and, in the future, literalString
+stringHelp = 
+    P.oneOf
+    [ P.succeed (identity)
+        |. P.token (P.Token "\\" ExpectingEscapeBackslash)
+        |= P.oneOf
+            [ P.map (\_ -> '\"') (P.token (P.Token "\"" ExpectingEscapeCharacter))
+            , P.map (\_ -> '\'') (P.token (P.Token "'"  ExpectingEscapeCharacter))
+            , P.map (\_ -> '\n') (P.token (P.Token "n"  ExpectingEscapeCharacter))
+            , P.map (\_ -> '\t') (P.token (P.Token "t"  ExpectingEscapeCharacter))
+            , P.map (\_ -> '\r') (P.token (P.Token "r"  ExpectingEscapeCharacter))
+            , P.succeed identity
+                |. P.token (P.Token "u" ExpectingEscapeCharacter)
+                |. P.token (P.Token "{" ExpectingUnicodeEscapeLeftBrace)
+                |= unicode
+                |. P.token (P.Token "}" ExpectingUnicodeEscapeRightBrace)
+            ]
+    ,  P.succeed identity
+        |= P.getChompedString (P.chompIf (always True) ExpectingChar)
+        |> P.andThen (\string ->
+                    string
+                        |> String.uncons
+                        |> Maybe.map (Tuple.first >> P.succeed)
+                        |> Maybe.withDefault (P.problem (CompilerBug "Multiple characters chomped in `literalChar`"))
+                )
+    ]
+
+
 literalChar : Parser_ Literal
 literalChar =
     (P.succeed identity
         |. P.symbol (P.Token "'" ExpectingSingleQuote)
-        |= P.getChompedString (P.chompIf (always True) ExpectingChar)
+        |= stringHelp
         |. P.symbol (P.Token "'" ExpectingSingleQuote)
     )
-        |> P.andThen
-            (\string ->
-                string
-                    |> String.uncons
-                    |> Maybe.map (Tuple.first >> Char >> P.succeed)
-                    |> Maybe.withDefault (P.problem (CompilerBug "Multiple characters chomped in `literalChar`"))
-            )
+    |> P.map (\n -> Char n)
```

Comment:
`|> P.map Char`

Reply:
Fixed by https://github.com/elm-in-elm/compiler/pull/20/commits/b7a3cf40afbfd735403bd6129e49426ac531887e

Thank you, @Janiczek!